### ISSUE TITLE
Grayscale, dtype and normalization

### ISSUE THREAD

**Issue by digit-jv** [Lornatang/RFB_ESRGAN-PyTorch#3]
the model currently does not support Grayscale images, he same applies to the loss function.
I could not find how to specify which normalization operation has to be applied to the dataset ([0,1], [-1,1], min max etc...)
Could you please get me some hints on how to proceed with the implementation.

**Comment by Lornatang** [comment]
@digit-jv Try modifly in_channels == 1 and replace Pixshuffle method.

**Comment by digit-jv** [comment]
while concerning the normalization?
It trains the PSNR model while it fails to trainthe GAN
`Exception has occurred: RuntimeError
all elements of input should be between 0 and 1
  File "\RFB_ESRGAN-PyTorch-main\train.py", line 239, in <module>
    errD_hr = adversarial_criterion(hr_output - torch.mean(sr_output), real_label)`

**Comment by Lornatang** [comment]
Can you print HR and SR shape？

**Comment by digit-jv** [comment]
`lr.shape
torch.Size([11, 3, 60, 60])
hr.shape
torch.Size([11, 3, 240, 240])`

**Comment by Lornatang** [comment]
You may need to modify the structure of the discriminator.
